packages/atomic/src/components/result-lists/result-list-common.tsx

```diff
@@ -235,38 +230,42 @@ export class ResultListCommon {
       return;
     }
 
+    const displayPlaceholders = !this.bindings.store.get('firstResultLoaded');
+
     const classes = this.getClasses(
       display,
       density,
       imageSize!,
       resultListState.firstSearchExecuted,
-      resultListState.isLoading
+      resultListState.isLoading,
+      displayPlaceholders
     );
     return (
       <Host>
         {templateHasError && <slot></slot>}
-        <div
-          class={`list-wrapper placeholder ${classes}`}
-          ref={setListWrapperRef}
-        >
+        <div class={`list-wrapper ${classes}`} ref={setListWrapperRef}>
           <ResultDisplayWrapper classes={classes} display={display}>
-            <ResultsPlaceholder
-              display={display}
-              density={density}
-              imageSize={imageSize}
-              resultsPerPageState={resultsPerPageState}
-            />
-            <Results
-              classes={classes}
-              bindings={this.bindings}
-              host={host}
-              display={display}
-              density={density}
-              imageSize={imageSize}
-              resultListState={resultListState}
-              resultListCommon={this}
-              getContentOfResultTemplate={getContentOfResultTemplate}
-            />
+            {displayPlaceholders && (
+              <ResultsPlaceholder
+                display={display}
+                density={density}
+                imageSize={imageSize}
+                resultsPerPageState={resultsPerPageState}
+              />
+            )}
```

Comment:
Placeholders were kept in the dom, lot's of unused elements in there, should help performance